spec.html

```diff
@@ -11978,7 +11980,7 @@ <h1>Runtime Semantics: GetTemplateObject ( _templateLiteral_ )</h1>
           1. Let _cookedStrings_ be TemplateStrings of _templateLiteral_ with argument *false*.
           1. Let _count_ be the number of elements in the List _cookedStrings_.
           1. Assert: _count_ &le; 2<sup>32</sup> - 1.
-          1. Let _template_ be ! ArrayCreate(_count_).
+          1. Let _template_ be ! ArrayCreate(_count_, %ArrayPrototype%, &laquo; [[TemplateLiteral]] &raquo;).
```

Comment:
What value does this slot have?

Reply:
It is initialized to undefined implicitly, and never updated. It's just a brand. The Intl spec does this a lot, for one.